src/pages/settings/Subscription/SubscriptionSettingsSection.tsx

```diff
@@ -0,0 +1,81 @@
+import {format} from 'date-fns';
+import React, {useMemo, useState} from 'react';
+import {View} from 'react-native';
+import Section from '@components/Section';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Navigation from '@navigation/Navigation';
+import ToggleSettingOptionRow from '@pages/workspace/workflows/ToggleSettingsOptionRow';
+import CONST from '@src/CONST';
+import ROUTES from '@src/ROUTES';
+
+function SubscriptionSettingsSection() {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const theme = useTheme();
+
+    // TODO these default state values will come from API in next phase
+    const [autoRenew, setAutoRenew] = useState(true);
+    const [autoIncrease, setAutoIncrease] = useState(false);
+
+    // TODO this will come from API in next phase
+    const expirationDate = format(new Date(), CONST.DATE.MONTH_DAY_YEAR_ABBR_FORMAT);
+
+    // TODO all actions will be implemented in next phase
+    const handleAutoRenewToggle = () => {
+        if (!autoRenew) {
+            // TODO make API call to enable auto renew here
+            setAutoRenew(true);
+            return;
+        }
+        Navigation.navigate(ROUTES.SETTINGS_SUBSCRIPTION_DISABLE_AUTO_RENEW_SURVEY);
+    };
+    const handleAutoIncreaseToggle = () => {
+        // TODO make API call to toggle auto increase here
+        setAutoIncrease(!autoIncrease);
+    };
+
+    const customTitle = useMemo(() => {
```

Comment:
Do we really need `useMemo` here? I see a pretty long list of dependencies, so how often this title is actually memorized?

Reply:
Dependencies are only styles and translate method. I doubt they will be changing a lot

Reply:
Okay, I agree. However, I still don't understand why we need `useMemo` here. There is no complex calculations, filtering, and so on.

Reply:
If you really wanted to this could be a separate component so I thought it might be worth memoizing. If you believe that cost of using memo is greater then performance we're saving here then I will remove it